pyaedt/modules/PostProcessor.py

```diff
@@ -4985,23 +4985,98 @@ def __init__(self, app):
+        geometry : str
+            Location to perform the calculation on. Options are
+            ``"Surface"`` and ``"Volume"``.
+        geometry_name : str or list of str
+            Objects to perform the calculation on. If a list is provided,
+            the calculation is performed on the combination of those
+            objects.
+        quantity : str
+            Quantity to compute.
+        normal : list of floats
+            Coordinate values for direction relative to normal. The default is ``""``,
+            in which case the normal to the face is used.
+        side : str, optional
+            String containing which side of the face to use. The default is
+            ``"Default"``. Options are ``"Adjacent"``, ``"Combined"``, and
+            `"Default"``.
+        mesh : str, optional
+            Surface meshes to use. The default is ``"All"``. Options are ``"All"`` and
+            ``"Reduced"``.
+        ref_temperature : str, optional
+            Reference temperature to use in the calculation of the heat transfer
+            coefficient. The default is ``"AmbientTemp"``.
+
+        Returns
+        -------
+        bool
+            ``True`` when successful, ``False`` when failed.
+        """
         if quantity not in AVAILABLE_QUANTITIES:
             raise AttributeError(
                 "Quantity {} is not supported. Available quantities are:\n{}".format(
                     quantity, ", ".join(AVAILABLE_QUANTITIES)
                 )
             )
+        if isinstance(normal, list):
+            if not isinstance(normal[0], str):
+                normal = [str(i) for i in normal]
+            normal = ",".join(normal)
+        if isinstance(geometry_name, str):
+            geometry_name = [geometry_name]
         self.calculations.append(
-            [entity, geometry, geometry_name, quantity, normal, side, mesh, ref_temperature, False]
+            [entity, geometry, ",".join(geometry_name), quantity, normal, side, mesh, ref_temperature, False]
         )  # TODO : last argument not documented
+        return True
 
     @pyaedt_function_handler()
-    def get_field_summary_data(self, sweep_name=None, design_variation={}, intrinsic_value="", pandas_output=False):
+    def get_field_summary_data(self, setup_name=None, design_variation={}, intrinsic_value="", pandas_output=False):
+        """
+        Get  field summary output computation.
+
+        Parameters
+        ----------
+        setup_name : str, optional
+            Setup name to use for the computation. The
+            default is ``None``, in which case the nominal variation is used.
+        design_variation : dict, optional
+            Dictionary containing the design variation to use for the computation.
+            The default is  ``{}``, in which case nominal variation is used.
+        intrinsic_value : str, optional
+            Intrinsic values to use for the computation. The default is ``""``.
```

Comment:
What behavior occurs when the default value is used. Format the answer as in the description for the previous parameter.

Reply:
```suggestion
            Intrinsic values to use for the computation. The default is ``""``, 
            suitable when no frequency needs to be selected.
```